base/applications/osk/settings.c

```diff
@@ -0,0 +1,103 @@
+﻿/*
+ * PROJECT:         ReactOS On-Screen Keyboard
+ * LICENSE:         GPL - See COPYING in the top level directory
+ * PURPOSE:         Settings file for warning dialog on startup
+ * COPYRIGHT:       Copyright 2018 Bișoc George (fraizeraust99 at gmail dot com)
+ */
+
+/* INCLUDES *******************************************************************/
+
+#include "settings.h"
+
+/* FUNCTIONS *******************************************************************/
+
+BOOL LoadDataFromRegistry(BOOL bIsEnabled)
```

Comment:
`bIsEnabled` is there so that I can initialize `Globals.bShowWarning` and setting it to TRUE, which you can see on `main.c` on line 120. Even though the behaviour works as it should be expected, I understand that the way it looks the code is a bit weird.

Reply:
So why don't you initialize it to `TRUE`, the constant, instead of using a parameter?